INSTALL.md

```diff
@@ -356,6 +356,10 @@ exec stack ghc --  \
 
 Don't forget to mark the file as `+x`: `chmod +x build`!
 
+Some example build scripts for `stack` and `cabal` are provided in the
+`scripts/build` directory of the `xmonad-contrib` distribution. You can
+see those online [here][].
```

Comment:
"here" is not a very descriptive link name, so it's better to just include the URI inline.

Reply:
Yeh, I'm also thinking I should have included some reasons one might want to use them.